lib/simplifyExpression/stepThrough.js

```diff
@@ -84,10 +84,10 @@ function step(node) {
     breakUpNumeratorSearch,
     // e.g. 3/x * 2x/5 => (3 * 2x) / (x * 5)
     multiplyFractionsSearch,
-    // e.g. (2x + 3)(x + 4) => 2x^2 + 11x + 12
-    distributeSearch,
     // e.g. abs(-4) => 4
     functionsSearch,
+    // e.g. (2x + 3)(x + 4) => 2x^2 + 11x + 12
```

Comment:
Can you give an example so I can see what you mean by this / why it's important?

Also, just thought of something: Does this PR conflict with your PR to expand exponents? E.g. for `(y + 1)^2 = 1`, do we want `y+1 = sqrt(1)` or `(y+1)(y+1) = 1`?